miden/tests/integration/operations/io_ops/env_ops.rs

```diff
@@ -143,10 +144,13 @@ fn caller() {
         end";
 
     // TODO: update and use macro?
+    let stack = StackInputs::try_from_values([1u64, 2, 3, 4, 5]).unwrap();
+    let inputs = ProgramInputs::none();
     let test = Test {
         source: program_source.to_string(),
         kernel: Some(kernel_source.to_string()),
-        inputs: ProgramInputs::from_stack_inputs(&[1, 2, 3, 4, 5]).unwrap(),
+        stack,
+        inputs,
         in_debug_mode: false,
     };
```

Comment:
nit: similar to the other comment, could this not be?:
```Rust
let test = Test {
    source: program_source.to_string(),
    kernel: Some(kernel_source.to_string()),
    stack: StackInputs::try_from_values([1u64, 2, 3, 4, 5]).unwrap(),
    inputs: ProgramInputs::none(),
    in_debug_mode: false,
};
```

Reply:
Fixed!